bHY6yh-d93162/cod-yyyiou_bHY6yh-d93162.xml

```diff
@@ -0,0 +1,250 @@
+              <lb ed="#C"/>sic secundas immoderate ferre leuitatis 
+              <lb ed="#C"/>est preclarum que est in omni uita et idem 
+              <lb ed="#C"/>semper uultus eademque frons</quote>
+            </p>
+            <p xml:id="bHY6yh-d93162-d1e235">
+              ¶ <ref xml:id="bHY6yh-d93162-Rd1e252">Idem 
+              <lb ed="#C"/>rethoricorum artis ueteris libro 2 capitulo 9</ref> <quote xml:id="bHY6yh-d93162-Qd1e239">Perseuerant<lb ed="#C"/>ia 
+              est in ratione bene considerata stabilis et perpetu<lb ed="#C"/>a 
+              mansio</quote>
+            </p>
+            <p xml:id="bHY6yh-d93162-d1e247">
+              ¶ <ref xml:id="bHY6yh-d93162-Rd1e266">Iacobus apostolus epistola I capitulo I</ref> <quote xml:id="bHY6yh-d93162-Qd1e251">Vir 
+              <lb ed="#C"/>duplex animo inconstans est in omnibus 
+              <lb ed="#C"/>uiis suis</quote>
+            </p>
+            <p xml:id="bHY6yh-d93162-d1e259">
+              ¶ <ref xml:id="bHY6yh-d93162-Rd1e278">Seneca epistolarum ad lucillum 
+              <lb ed="#C"/>libro II epistola I</ref> <quote xml:id="bHY6yh-d93162-Qd1e252">Primum argumentum com<lb ed="#C"/>posite 
+              mentis existimo posse consistere 
+              <lb ed="#C"/>et secum morari non conualescit planta 
+              etc.</quote>
+            </p>
+            <p xml:id="bHY6yh-d93162-d1e268"> 
+              ¶ <ref xml:id="bHY6yh-d93162-Rd1e294">Idem libro 5 epistola 5</ref> <quote xml:id="bHY6yh-d93162-Qd1e270">Ante omnia hoc cura 
+              <lb ed="#C"/>ut constes tibi</quote>
+            </p>
+            <p xml:id="bHY6yh-d93162-d1e276"> 
+              ¶ <ref xml:id="bHY6yh-d93162-Rd1e304">Idem libro 7 epistola 4</ref> 
+              <lb ed="#C"/><quote xml:id="bHY6yh-d93162-Qd1e278">Stultitia nichil constat nichil diu 
+              <lb ed="#C"/>placet</quote>
+            </p>
+            <p xml:id="bHY6yh-d93162-d1e284"> 
+              ¶ <ref xml:id="bHY6yh-d93162-Rd1e315">Idem de tranquillitate animi</ref> <quote xml:id="bHY6yh-d93162-Qd1e286">humanus 
+              <lb ed="#C"/>animus agilis est et pronus ad motus semper 
+              <lb ed="#C"/>presentibus infesta est leuitas</quote>
+            </p>
+            <p xml:id="bHY6yh-d93162-d1e297"> 
+              ¶ <ref xml:id="bHY6yh-d93162-Rd1e330">Idem de 
+              <lb ed="#C"/>consolatione ad elbiam matrem</ref> <quote xml:id="bHY6yh-d93162-Qd1e299">Mobilis et inqui<lb ed="#C"/>eta 
+              mens homini data est nunquam se tenere 
+              <lb ed="#C"/>potest spargitur et cogitationes suas in omnia 
+              <lb ed="#C"/>nota atque ignota dimittit vaga inquie<lb ed="#C"/>ta 
+              et impatiens et nouitate rerum le<lb ed="#C"/>tissima</quote>
+            </p>
+            <p xml:id="bHY6yh-d93162-d1e313"> 
+              <ref xml:id="bHY6yh-d93162-Rd1e349"></ref> <quote xml:id="bHY6yh-d93162-Qd1e315"></quote>
+            </p>
```

Comment:
What's going on here. Is this paragraph "omitted"? 

If so, let's delete the ref and quote, but leave the paragraph and inside the paragraph we should write [omitted] so I know that this paragraph is omitted in this paragraph

```suggestion
            <p xml:id="bHY6yh-d93162-d1e313"> 
            [omitted]
            </p>
```